packages/react-composites/src/composites/MeetingComposite/adapter/AzureCommunicationMeetingAdapter.ts

```diff
@@ -454,6 +464,42 @@ export const createAzureCommunicationMeetingAdapter = async ({
   return new AzureCommunicationMeetingAdapter(callAdapter, chatAdapter);
 };
 
+/**
+ * Arguments for {@link createAzureCommunicationMeetingAdapterFromClient}
+ *
+ * @beta
+ */
+export type AzureCommunicationMeetingAdapterFromClientArgs = {
+  meetingLocator: CallAndChatLocator | TeamsMeetingLinkLocator;
```

Comment:
meeting locator isn't quite applicable to this construction as it requires a chatThreadId but we don't actually use the chatThreadId here (the chatThreadClient that is passed in here uses that).
Instead just make this a `callLoctor: CallLocator | TeamsMeetingLinkLocator`